### PR TITLE
Removed intermediary handler for width changed event

### PR DESCRIPTION
**Pull request by MitchelPaulin**
A small proposed change here, thought it was strange that we were changing the property directly on a web component. This just moves the event handler into the web component that cares about the property change and makes the flow a bit easier to follow.

### CODE REVIEW

File: src/components/editor/rubric-levels-editor.js [hunk 1]
```diff
@@ -189,7 +189,7 @@ class RubricLevelsEditor extends composeMixins(
 		requestAnimationFrame(() => {
 			const section = this.shadowRoot.querySelector('#levels-section');
 			if (section && section.offsetWidth) {
-				this.dispatchEvent(new CustomEvent('d2l-rubric-editor-levels-width-changed', {
+				window.dispatchEvent(new CustomEvent('d2l-rubric-editor-levels-width-changed', {
```
Comment (devpow112) [inline src/components/editor/rubric-levels-editor.js:192]: Does using `this.` still not work here? Usually you want to keep using `this` where possible.

Comment (MitchelPaulin) [reply src/components/editor/rubric-levels-editor.js:192]: Yes, when publishing to `this.` the event wasn't being received by the criteria editor

File: src/components/editor/rubric-criteria-editor.js [hunk 2]
```diff
@@ -187,6 +187,20 @@ class RubricCriteriaEditor extends composeMixins(
 		this.partialEditMode = false;
 		this.useNewBrightspaceEditor = false;
 	}
+	connectedCallback() {
+		super.connectedCallback();
+		window.addEventListener('d2l-rubric-editor-levels-width-changed', (e) => {
```
Comment (devpow112) [inline src/components/editor/rubric-criteria-editor.js:192]: If you switch this to use `this` and does the dispatch work with `this` then?

Comment (MitchelPaulin) [reply src/components/editor/rubric-criteria-editor.js:192]: No, but I checked and dispatch the event can use `this.`, but the listeners need to be listening to `window`

**Comment by MitchelPaulin** [comment]
Vdiff changes are from resizing issues, though most work, oddly enough its only the analytic points rubrics which are failing, will look into this further if I have some extra time

**Pull request closed without merging**